DSC/dsc.py

```diff
@@ -327,7 +340,8 @@ def get_openssl_version():
     else:
         error_msg = 'This system does not have a supported version of OpenSSL installed. Supported version: 0.9.8*, 1.0.*'
```

Comment:
Currently, I am just improving the tracing and get hands on the release cycle. In the next iteration, I will test this if DSC bits can support 1.1.*